File: LeetCode/Easy/0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
    	int[] result = null;
     	HashMap<Integer, Integer> numsMap = new HashMap<Integer, Integer>();
    	for (int i = 0; i < nums.length; i++) {
    		numsMap.put(nums[i], i);
    	}
    	

    	for (int i = 0; i < nums.length; i++) {
    		int contrast = target - nums[i];
    		if (numsMap.containsKey(contrast) && i != numsMap.get(contrast)) {
    			result = new int[]{i, numsMap.get(contrast)};
    		}
    	}
    	
    	return result;
    }
}

File: LeetCode/Easy/0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums)):
            try:
                idx = nums.index(target - nums[i])
                if idx >= 0 and i != idx:
                    return [i, idx]
            except:
                pass